app/models/attribute/ConfigTable.scala

```diff
@@ -0,0 +1,204 @@
+  def southwestBoundaryLat: Column[Double] = column[Double]("southwest_boundary_lat", O.NotNull)
+  def southwestBoundaryLng: Column[Double] = column[Double]("southwest_boundary_lng", O.NotNull)
+  def northeastBoundaryLat: Column[Double] = column[Double]("northeast_boundary_lat", O.NotNull)
+  def northeastBoundaryLng: Column[Double] = column[Double]("northeast_boundary_lng", O.NotNull)
+  def defaultMapZoom: Column[Double] = column[Double]("default_map_zoom", O.NotNull)
+  def tutorialStreetEdgeID: Column[Int] = column[Int]("tutorial_street_edge_id", O.NotNull)
+  def offsetHours: Column[Int] = column[Int]("update_offset_hours", O.NotNull)
+  def excludedTags: Column[String] = column[String]("excluded_tags", O.NotNull)
+  def apiAttributeCenterLat: Column[Double] = column[Double]("api_attribute_center_lat", O.NotNull)
+  def apiAttributeCenterLng: Column[Double] = column[Double]("api_attribute_center_lng", O.NotNull)
+  def apiAttributeZoom: Column[Double] = column[Double]("api_attribute_zoom", O.NotNull)
+  def apiAttributeLatOne: Column[Double] = column[Double]("api_attribute_lat1", O.NotNull)
+  def apiAttributeLngOne: Column[Double] = column[Double]("api_attrubute_lng1", O.NotNull)
+  def apiAttributeLatTwo: Column[Double] = column[Double]("api_attribute_lat2", O.NotNull)
+  def apiAttributeLngTwo: Column[Double] = column[Double]("api_attrubute_lng2", O.NotNull)
+  def apiStreetCenterLat: Column[Double] = column[Double]("api_street_center_lat", O.NotNull)
+  def apiStreetCenterLng: Column[Double] = column[Double]("api_street_center_lng", O.NotNull)
+  def apiStreetZoom: Column[Double] = column[Double]("api_street_zoom", O.NotNull)
+  def apiStreetLatOne: Column[Double] = column[Double]("api_street_lat1", O.NotNull)
+  def apiStreetLngOne: Column[Double] = column[Double]("api_street_lng1", O.NotNull)
+  def apiStreetLatTwo: Column[Double] = column[Double]("api_street_lat2", O.NotNull)
+  def apiStreetLngTwo: Column[Double] = column[Double]("api_street_lng2", O.NotNull)
+  def apiRegionCenterLat: Column[Double] = column[Double]("api_region_center_lat", O.NotNull)
+  def apiRegionCenterLng: Column[Double] = column[Double]("api_region_center_lng", O.NotNull)
+  def apiRegionZoom: Column[Double] = column[Double]("api_region_zoom", O.NotNull)
+  def apiRegionLatOne: Column[Double] = column[Double]("api_region_lat1", O.NotNull)
+  def apiRegionLngOne: Column[Double] = column[Double]("api_region_lng1", O.NotNull)
+  def apiRegionLatTwo: Column[Double] = column[Double]("api_region_lat2", O.NotNull)
+  def apiRegionLngTwo: Column[Double] = column[Double]("api_region_lng2", O.NotNull)
+  def databaseName: Column[String] = column[String]("database_name", O.NotNull)
+
+  def * = (openStatus, mapathonEventLink, cityCenterLat, cityCenterLng, southwestBoundaryLat, southwestBoundaryLng, northeastBoundaryLat, northeastBoundaryLng, defaultMapZoom, tutorialStreetEdgeID, offsetHours, excludedTags, databaseName, (
```

Comment:
This section looks more verbose than I remember it needing to be, but I never actually had to do this, and I guess it doesn't really matter how verbose it look 😅